jobs/ci-kubernetes-e2e-gce-cosdev-k8sbeta-default.env

```diff
@@ -0,0 +1,22 @@
+# AUTO-GENERATED - DO NOT EDIT.
```

Comment:
No need to fix as part of this commit, but a pet peeve of mine is seeing headers like this that don't tell you what script actually does the generation - it drives me batty when I'm digging into code ! Consider hardcoding the script name in here ...

```
# AUTO-GENERATED by my-awesome-script.py - DO NOT EDIT
```